packages/edit-site/src/components/sidebar-navigation-screen-template/index.js

```diff
@@ -119,7 +114,7 @@ export default function SidebarNavigationScreenTemplate() {
 						postId={ postId }
 						toggleProps={ { as: SidebarButton } }
 						onRemove={ () => {
-							navigator.goTo( POST_TYPE_PATH[ postType ] );
+							navigator.goTo( postType );
```

Comment:
I'm not clear on this bit. I would expect to see a path — so, at the very least, `'/' + postType`. Can you explain to me what this does?

Reply:
Nice catch, fixed at https://github.com/WordPress/gutenberg/pull/60667/commits/0054ab40041cb7c4ba645ae782a65ed768738962